lib/trento/hosts/projections/host_read_model.ex

```diff
@@ -32,6 +32,7 @@ defmodule Trento.Hosts.Projections.HostReadModel do
     field :provider, Ecto.Enum, values: Provider.values()
     field :provider_data, :map
     field :saptune_status, :map
+    field :prometheus_targets, :map
```

Comment:
considering we don't have a default in the read model I think we could omit in the aggregate